src/D2L.CodeStyle.Analysis/MutabilityInspector.cs

```diff
@@ -128,14 +137,8 @@ ITypeSymbol type
 					continue;
 				}
 				if( member is IMethodSymbol ) {

```

Comment:
The contents of this foreach loop field like a method in itself.  That way we can have less control statements